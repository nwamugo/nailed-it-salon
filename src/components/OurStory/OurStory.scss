.ourstory-box {
  width: 100%;
  padding: 0;
  margin: 0;
  margin-top: 7%;
}
.ourstory-board {
  width: 70%;
  margin: 0 auto;
  padding: 0;
  text-align: center;
  background-color: #000000;
}
.story {
  font-size: 3rem;
  font-family: "Roboto", sans-serif;
  text-align: right;
  text-justify: inter-word;
  color: #ffffff;
  margin-top: 20px;
  margin-left: 4%;
  margin-right: 4%;
}
.our-story-image {
  background-color: green;
  margin: 0 auto;
  background: url(../../shared/images/ourstory.jpg);
  background-size: cover;
  background-repeat: no-repeat;
  @media (min-width: 320px) and (max-width: 480px) {
    display: none;
  }
  @media (min-width: 481px) and (max-width: 767px) {
    display: none;
  }
  @media (min-width: 768px) and (max-width: 1024px) {
    display: none;
  }
}

.story-container {
  margin: 7rem 0;
  align-items: center;
  display: flex;
  align-content: center;
  flex-direction: column;
}

.story-container .primary-header {
  text-transform: uppercase;
  font-family: "Josefin Sans", sans-serif;
  font-size: 6rem;
  font-weight: 600;
  letter-spacing: 0.04em;
  color: #e294c5;
}
