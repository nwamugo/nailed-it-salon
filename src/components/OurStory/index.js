import React from 'react';
import { useTranslation } from 'react-i18next';
import './OurStory.scss';

const OurStory = () => {
  const { t } = useTranslation();
  return (
    <div className="ourstory-box" id="ourstory">
      <div className="ourstory-board row">
        <div className="our-story-image col" />
        <div className="col">
          <div className="story-container">
            <h2 className="primary-header">{t('header.story')}</h2>
            <p className="story">
              {t('story.part1')}
              <br />
              <br />
              {t('story.part2')}
            </p>
          </div>
        </div>
      </div>
    </div>
  );
};

export default OurStory;
