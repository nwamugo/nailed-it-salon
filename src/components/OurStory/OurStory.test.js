import React from 'react';
import { shallow } from 'enzyme';
import OurStory from './index';

describe('OurStory', () => {
  it('should not have regressed', () => {
    const component = shallow(<OurStory />);

    expect(component).toMatchSnapshot();
  });
});
