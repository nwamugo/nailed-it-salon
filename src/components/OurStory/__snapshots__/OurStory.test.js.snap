// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurStory should not have regressed 1`] = `
<div
  className="ourstory-box"
  id="ourstory"
>
  <div
    className="ourstory-board row"
  >
    <div
      className="our-story-image col"
    />
    <div
      className="col"
    >
      <div
        className="story-container"
      >
        <h2
          className="primary-header"
        >
          header.story
        </h2>
        <p
          className="story"
        >
          story.part1
          <br />
          <br />
          story.part2
        </p>
      </div>
    </div>
  </div>
</div>
`;
